src/com/asjaHas/qamp/oophomework3/Task2/Engine.java

```diff
@@ -0,0 +1,41 @@
+package com.asjaHas.qamp.oophomework3.Task2;
+
+public class Engine {
+    protected double capacity;
+
+    public Engine(final double capacity) {
+        this.capacity = capacity;
+    }
+
+    public double getCapacity() {
+        return capacity;
+    }
+
+    public void setCapacity(final double capacity) {
+        this.capacity = capacity;
+    }
+
+    public double increaseCapacity(final double increaseCapacity) {
+        this.capacity += increaseCapacity;
+        return capacity;
+    }
+
+    public double decreaseCapacity(final double decreaseCapacity) {
+        this.capacity -= decreaseCapacity;
+        return capacity;
+    }
+
+    public double consumeCapacity(final double speed) {
```

Comment:
this is very good, using these helper methods, one thing, you could have done is make use of `decreaseCapacity` like: 

`decreaseCapacity(0.2 * speed)`